presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/PropertyDerivations.java

```diff
@@ -183,13 +183,23 @@ public ActualProperties visitEnforceSingleRow(EnforceSingleRowNode node, List<Ac
         public ActualProperties visitAssignUniqueId(AssignUniqueId node, List<ActualProperties> inputProperties)
```

Comment:
The commit message says:

>   - remove grouped and sorting local properties when passing through a cross join.

but I don't see where that was changed in the diff.

Reply:
@martint 

I extracted this change and `preserve local properties on the probe columns for the output of the left join` into separate commits, but forgot to update the commit message. Will do.

* `Update AddLocalExchanges to plan streaming aggregations based on properties` commit includes changes for the cross join
* `Preserve local properties of the probe side when passing through a LEFT join` commit includes changes for the left join

Note: there are no unit tests for `PropertyDerivations` and therefore it is not possible to test behaviors that don't manifest themselves in the final query plans. Before updating `AddLocalExchanges` to use `partitioned_on` property for planning streaming aggregations, I couldn't come up with a query whose plan would be affected by that bug. Hence, the fix appears in the same commit as the changes to `AddLocalExchanges`.